addons/mail/static/src/core/common/core.scss

```diff
@@ -157,3 +157,8 @@ a.o-discuss-mention {
         opacity: 100 !important;
     }
 }
+
+.o-discuss-has-size-constraints {
+    width: Min(95vw, 400px);
+    max-height: Min(calc(100vh - 140px), 530px);
+}
```

Comment:
Semantics of the classname is quite poor.

It was a technical detail of the Component, as a classname modifier, so it was not a huge deal. But now this is global, the naming and purpose should be clearer.

From my understanding of the code, this is a popover size limiter for discuss thread actions? We could name it `o-mail.Discuss-threadActionPopover`?

Reply:
Agreed, I just factorize this as two classes were doing the same without giving it much thought. This name is indeed better + it enforces coherence between action popover.